Multisig/UI/Safe Management/Create Safe/CreateSafeFlow.swift

```diff
@@ -113,6 +113,7 @@ class CreateSafeFlow: UIFlow, ASAuthorizationControllerPresentationContextProvid
             let view = self.factory.safeCreatingViewController()
             view.onSuccess = { [weak self] in
```

Comment:
I think this is wrong, 

As it's presented using "show" function then is should be dismissed when the flow stop function and dismiss should have no effect on it,

Reply:
second that